CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # CHANGELOG
 
+## VNext 
+- Updating URL validation
```

Comment:
The description should be in a completed form. Not `Updating` but `Updated`.

Reply:
Done